README.md

```diff
@@ -0,0 +1,2444 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Pls remove the contents of this file as it contains some default content from the boilerplate repo.
Add any content relevant to our if you thought of any.